spec/models/user_spec.rb

```diff
@@ -13,6 +13,31 @@
     it { is_expected.to validate_presence_of(:provider) }
 
     it { is_expected.to validate_presence_of(:email) }
+    it { is_expected.to validate_presence_of(:password_confirmation) }
     it { is_expected.to validate_uniqueness_of(:email).scoped_to(:provider).case_insensitive }
+
+    context 'password confirmation' do
+      let(:valid_user) do
+        {
+          name: Faker::Name.name,
+          email: Faker::Internet.email,
+          provider: 'greenlight',
+          password: 'Password123+',
+          password_confirmation: 'Password123+'
+        }
+      end
```

Comment:
No need for this since you have the user_factory

Reply:
Makes since after using `build`.

Reply:
Done,